### ISSUE TITLE
native extension not showing up automatically

### ISSUE THREAD

**Issue by Scoppio** [nightblade9/godot-nvda-integration#4]
When adding the plugin into my project it failed hard for being unable to find the NVDA ref.

Looking into your demo project I saw that you have the nvda extension in the native_extensions, but there is nothing telling on how this got there, the project didnt add it automatically in any way for me (I added the gdextension to the root of the project as pointed in the readme), so I went on and edited it in the godot project of mine and it now reads nvda as a thing. which is fine, but it may not be working out of the box when using Godot 4.3 (my case)

**Comment by nightblade9** [comment]
Hey, thanks for letting me know about this! I haven't used the addon with Godot 4.3 yet, and don't have any planned projects that use it. Could I trouble you to open a PR with some sort of fix for this? Maybe just a README update for now indicating what you need to do, would be enough for now.
